hoomd/mpcd/test-py/stream_sphere_test.py

```diff
@@ -0,0 +1,197 @@
+
+import unittest
+import numpy as np
+import hoomd
+from hoomd import md
+from hoomd import mpcd
+
+# unit tests for mpcd sphere streaming geometry
+class mpcd_stream_sphere_test(unittest.TestCase):
+    def setUp(self):
+        # establish the simulation context
+        hoomd.context.initialize()
+
+        # default testing configuration
+        hoomd.init.read_snapshot(hoomd.data.make_snapshot(N=0, box=hoomd.data.boxdim(L=10.)))
+
+        # initialize the system from the starting snapshot
+        snap = mpcd.data.make_snapshot(N=4)
+        """
+        particle 1: Hits the wall in the second streaming step, gets reflected accordingly
+        particle 2: Always inside the sphere, so no reflection by the BC
+        particle 3: Hits the wall normally and gets reflected back.
+        particle 4: Lands almost exactly on the sphere surface and needs to be backtracked one complete step
+        """
```

Comment:
We use `#` even for multiline comments in this code:
```suggestion
        # particle 1: Hits the wall in the second streaming step, gets reflected accordingly
        # particle 2: Always inside the sphere, so no reflection by the BC
        # particle 3: Hits the wall normally and gets reflected back.
        # particle 4: Lands almost exactly on the sphere surface and needs to be backtracked one complete step
```